docs/cedarling/cedarling-policy-store.md

```diff
@@ -206,12 +206,8 @@ The Token Entity Metadata Schema defines how tokens are mapped, parsed, and tran
 
 - **role_mapping**: (String OR Array of String, *Optional*) Indicates which field in the token should be used for role-based access control. If not needed, set to an empty string (`""`).
 
-You can include a `role_mapping` in each token but only the first one that get parsed will be recognized by Cedarling. Cedarling parses the `role_mapping`s for each token in this order:
-
-1. `access_tokens`
-2. `id_tokens`
-3. `userinfo_tokens`
-4. `tx_tokens`
+You can include a `role_mapping` in each token, all of them will be executed by Cedarling.
+If none `role_mapping` defined the `Cedarling` will try to find role in `userinfo` token in field `role`.
```

Comment:
Shouldn't Cedarling error if the `role` claims are different for each token instead of trying to authorize with each one? This seems insecure since they can effectively *try* multiple values for the `role` in each request -- and if the request should be denied using the `access_token` because of a wrong `role`, Cedarling could still allow the request to go through as long as the `role` is correct for the other tokens.

Reply:
it is OK if in different tokens we have different roles otherwise it has no sense to have mapping role from many tokens

Reply:
> it has no sense to have mapping role from many tokens
Correct me if I'm wrong but I thought that the reason why we enable mapping from multiple tokens is because some IDPs do not provide all the tokens. for example, some IDPs might only give ID tokens. Some might also only give Userinfo Tokens.

Reply:
At first, I have implemented role mapping only from one token. And @SafinWasi and @nynymike ask me to extend this to get roles from many tokens.

Reply:
This is probably a corner case. In Jans Auth Server for example, the Userinfo and id_token claims should be the same (unless there is an interception script).